src/test/java/org/synyx/urlaubsverwaltung/absence/ArchitectureTest.java

```diff
@@ -0,0 +1,67 @@
+package org.synyx.urlaubsverwaltung.absence;
+
+import com.tngtech.archunit.core.domain.JavaClass;
+import com.tngtech.archunit.core.domain.JavaClasses;
+import com.tngtech.archunit.core.importer.ClassFileImporter;
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.tngtech.archunit.core.importer.ImportOptions;
+import com.tngtech.archunit.library.dependencies.SliceAssignment;
+import com.tngtech.archunit.library.dependencies.SliceIdentifier;
+import com.tngtech.archunit.library.dependencies.SliceRule;
+import com.tngtech.archunit.library.dependencies.SlicesRuleDefinition;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+
+public class ArchitectureTest {
+
+    private ImportOptions options = new ImportOptions().with(ImportOption.Predefined.DO_NOT_INCLUDE_JARS)
+            .with(ImportOption.Predefined.DO_NOT_INCLUDE_ARCHIVES).with(ImportOption.Predefined.DO_NOT_INCLUDE_TESTS);
+
+    private JavaClasses classes = new ClassFileImporter(options).importPackages("org.synyx.urlaubsverwaltung");
+
+    @Test
+    @Ignore("does not run successfully, yet")
```

Comment:
Da nun das einzige was archunit macht auf @Ignore gesetzt ist, sollte das eher nicht in den Branch gemerged werden. Wir nehmen sonst nur Dependencies dazu und verwenden diese nicht oder sehe ich das falsch?